### PR TITLE
Coopr 569 - ssl\tls support added

### PR DESCRIPTION
**Pull request by kpovetkin**
according to https://issues.cask.co/browse/COOPR-569

### CODE REVIEW

**Comment by gaarf** [comment]
please match existing indentation! We use http://editorconfig.org/

File: coopr-standalone/bin/coopr.sh [hunk 1]
```diff
@@ -154,12 +154,6 @@ COOPR_PROTOCOL=http
 export COOPR_SSL=`read_property server.ssl.enabled ${COOPR_SERVER_CONF}/coopr-site.xml`
 if [ "${COOPR_SSL}" == "true" ]; then
   COOPR_PROTOCOL=https
-
-  COOPR_NODEJS_SSL_PATH=`read_property server.nodejs.ssl.path ${COOPR_SERVER_CONF}/coopr-security.xml`
-  COOPR_NODEJS_SSL_KEY_FILENAME=`read_property server.nodejs.ssl.key ${COOPR_SERVER_CONF}/coopr-security.xml`
-  export COOPR_NODEJS_SSL_KEY=$COOPR_NODEJS_SSL_PATH/$COOPR_NODEJS_SSL_KEY_FILENAME
-  COOPR_NODEJS_SSL_CRT_FILENAME=`read_property server.nodejs.ssl.crt ${COOPR_SERVER_CONF}/coopr-security.xml`
-  export COOPR_NODEJS_SSL_CRT=$COOPR_NODEJS_SSL_PATH/$COOPR_NODEJS_SSL_CRT_FILENAME
```
Comment (wolf31o2) [inline coopr-standalone/bin/coopr.sh]: Why is all of this being removed?

Comment (kpovetkin) [inline coopr-standalone/bin/coopr.sh]: There was a discussion that these parameters should not be in config files.

**Comment by ashvayka** [comment]
@gaarf  We believe travis issues are not related to changes. Please launch build manually once again.

**Comment by jwang47** [comment]
Please try merging from develop. I ran the develop unit tests and they passed.

**Comment by gaarf** [comment]
Discussed internally with @wolf31o2 - we have decided to implement this ourselves. As it stands the PR is mostly whitespace differences and adding a bunch of single-use variables.

**Pull request closed without merging**